tests/setup/setup.go

```diff
@@ -36,6 +36,7 @@ func SetupDI(t fxtest.TB, option fx.Option) (context.Context, context.CancelFunc
 			fx.Populate(&middleware),
 			fx.Populate(&route),
 		),
+		fx.NopLogger,
```

Comment:
@Cyantosh0 Why was `fx.NopLogger` added?

Reply:
@dipeshdulal sir, I think, displaying all the dependency injection logs in testing is not good. So, I added `fx.NopLogger` to just show the test cases in the terminal.